lib/mosaic-objects/src/main/java/org/eclipse/mosaic/lib/objects/v2x/GenericV2xMessage.java

```diff
@@ -63,6 +64,12 @@ public GenericV2xMessage(MessageRouting routing, ToDataOutput messagePayload, lo
         this.payload = new EncodedPayload(messagePayload, minimalMessageSize);
     }
 
+    public GenericV2xMessage(GenericV2xMessage message) {
```

Comment:
I don't see any usages of this new constructor. If we don't need it please remove it.

Reply:
done